src/components/breadcrumbs/BreadcrumbLink.tsx

```diff
@@ -3,13 +3,25 @@ import { Svg } from '../svg/Svg';
 
 export interface IBreadcrumbLinkProps {
   name: string;
-  link: string;
+  link?: string;
+  onClick?: (props: IBreadcrumbLinkProps) => boolean;  // return false to cancel the href event
 }
 
 export class BreadcrumbLink extends React.Component<IBreadcrumbLinkProps, {}> {
+
+  private onClick(e: React.MouseEvent<HTMLAnchorElement>): boolean {
+    if (this.props.onClick && !this.props.onClick(this.props)) {
+      e.stopPropagation();
+      e.nativeEvent.stopImmediatePropagation();
+      e.preventDefault();
+      return false;
+    }
+    return true;
+  }
```

Comment:
not too sure why you have that here, could you explain it in plain english or french

Reply:
thinking about it twice, I think a better approach in this case is to use a `span` instead of an `a` tag to prevent the issue you are trying to prevent here.  

if a link, use:
```  
     return (<li className='breadcrumb-title'>
       <a className='link' href={this.props.link}>{this.props.name}</a>
       <Svg svgName='arrow-right-rounded' svgClass='breadcrumb-arrow' />
     </li>);
```  
  
if an `onClick` is used:
```  
     return (<li className='breadcrumb-title'>
       <span className='link' onClick={(e: React.MouseEvent<HTMLAnchorElement>) => this.onClick(e)}>{this.props.name}</span>
       <Svg svgName='arrow-right-rounded' svgClass='breadcrumb-arrow' />
     </li>);  
```

Reply:
Je ne partage pas vraiment ton idée du span au lieu du a, je crois qu'il est logique que le tag reste un a puisqu'on veut navigater la majorité du temps mais on peut tjs ajouter un onClick dans certains use case spécifique.  on a juste a handler les events et retourner false ou true au onClick si on veut permettre le href ou pas. De cette façon, tout les cas son possible et sa permet au user de faire ce qu'il veut avec le href et le onClick